advanced_communication_options/update_comms_settings.py

```diff
@@ -125,8 +196,8 @@ def check_pi3():
 	if hw_id=="BCM2709":
```

Comment:
Huh, so you're saying that the Raspberry Pi 3 has 2 chips depending on date that was produced?
Like it has the `BCM2709`/`BCM2835`?